lib/pbench/test/unit/agent/test_tool_meister.py

```diff
@@ -57,37 +82,37 @@ class TestCreateTar:
         "tools": [],
     }
 
-    def test_create_tar(self, agent_setup, mock_tar):
+    def test_create_tar(self, agent_setup, mock_tar, caplog):
         """Test create tar file"""
+        self.logger.setLevel(level=logging.ERROR)
         tm = ToolMeister(
             pbench_install_dir=None,
             tmp_dir=None,
             tar_path="tar_path",
             sysinfo_dump=None,
-            params=TestCreateTar.tm_params,
+            params=self.tm_params,
             redis_server=None,
             logger=None,
         )
         tmp_dir = agent_setup["tmp"]
-        tar_file = "test.tar.xz"
-
         cp = tm._create_tar(tmp_dir, pathlib.Path(tar_file))
         assert cp.returncode == 0
         assert cp.stdout == b""
 
-    def test_create_tar_ignore_warnings(self, agent_setup, mock_tar_no_warnings):
+    def test_create_tar_ignore_warnings(
+        self, agent_setup, mock_tar_no_warnings, caplog
+    ):
         """Test if we can suppress the errors raised during the tar creation"""
```

Comment:
The docstring isn't that useful, and you've repeated it in multiple test cases.

In any case, "suppress[ing] the errors" shouldn't really be the point of any of the unit tests; that's between you, your code, and pytest. What we want to actually test is the tool meister send's retry logic. Describing the purpose and strategy of each test is much more useful.